core/domain/base_model_validators.py

```diff
@@ -33,6 +33,10 @@
 
 USER_ID_REGEX = 'uid_[a-z]{32}'
 
+# These are model ids used for oppia bots like suggestion bot and migration
```

Comment:
Drop "oppia" -- easy to confuse with oppiabot on the dev repo.

Reply:
Done.

Reply:
Not done?

Reply:
Done.